README.md

```diff
@@ -13,8 +13,9 @@ that they are not required to be valid UTF-8.
 
 https://docs.rs/bstr
 
-The documentation includes a section on why you might want to use this crate.
+### Why use BString over String?
 
+Read this part of the documentation https://docs.rs/bstr/0.1.0/bstr/#when-should-i-use-byte-strings.
```

Comment:
Could you please wrap this to 79 columns inclusive? (Just put the URL on its own line.)